src/main/java/org/project36/qualopt/service/dto/StudyInfoDTO.java

```diff
@@ -0,0 +1,20 @@
+package org.project36.qualopt.service.dto;
+
+import org.project36.qualopt.domain.Study;
+
+public class StudyInfoDTO {
```

Comment:
Interesting, what else do you envision going into this StudyInfoDTO?

Reply:
Issue #42 referenced also having an upload feature for uploading pdf/documents. I also envisioned other static site related things going into this DTO as well. Also, REST API for retrieving study entity requires authentication and didn't want to break that security. so instead added a security exception for retrieving a study info object as this needs to be viewed by Participants who won't be authenticated

Reply:
Yeah, that makes sense! Is there some Maven dependency (or just base java library) that can facilitate storage of PDFs? Not 100% sure if the current DB architecture would just be able to store them